### PR TITLE
creación de organización omolds

### PR DESCRIPTION
**Pull request by omolds**
adjuntando organización de omolds

### CODE REVIEW

**Comment by omolds** [comment]
I signed it!

**Comment by omolds** [comment]
I signed it!

**Comment by omolds** [comment]
I signed it!

File: organizations/omolds.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "Omolds",
```
Comment (pfrisella) [inline organizations/omolds.json]:
```suggestion
  "id": "OMOLDS",
```

**Comment by omolds** [comment]
CLA for commit author signed

**Comment by omolds** [comment]
I signed it!

**Comment by omolds** [comment]
I signed it!

**Comment by omolds** [comment]
I signed it!

**Comment by omolds** [comment]
I signed it

**Comment by pfrisella** [comment]
CLA issue needs to be resolved.
Closing due to inactivity.

**Pull request closed without merging**